module3/projects/rails_engine/requirements.md

```diff
@@ -93,17 +99,86 @@ module Api
 end
 ```
 
-## 3a. ReST Endpoints
+#### Error Responses
+
+If the user causes an error for which you are sending a 400-series error code, the JSON body of the response should follow a similar JSON Spec of a `data` element, with a `nil` ID, and empty attributes.
+
+As an **EXTENSION**, customize the error message to use this format instead:
```

Comment:
Do we want this extension to also follow the JSON Spec with a `data` and have message and errors be nested in attributes?

Reply:
No, typical APIs would have a completely different structure for error messages